assets/js/elife-loader.js

```diff
@@ -1,15 +1,33 @@
 (function (window) {
```

Comment:
Don't run this through an IIFE, as could make testing more difficult.

Reply:
Completed